apitest/docker/bitcoind-entrypoint.sh

```
@@ -0,0 +1,82 @@
+  elif command -v bash >/dev/null 2>&1; then
+    bash -c "exec 3<>/dev/tcp/${host}/${port}; printf '%s\n' '${hash}' >&3; exec 3<&-" >/dev/null 2>&1 || true
+  fi
+done
+EOF
+chmod +x /usr/local/bin/blocknotify.sh
+
+# Create a descriptor wallet "testwallet" after RPC comes up. The legacy
+# wallet.dat shipped in dao-setup.zip is unloadable by bitcoind 29; tests just
+# need ANY wallet for generatetoaddress, so we make a fresh descriptor one.
+(
+  for i in $(seq 1 60); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)